app/src/androidTest/java/io/reark/rxgithubapp/activities/utils/SystemAnimations.java

```diff
@@ -1,9 +1,5 @@
 package io.reark.rxgithubapp.activities.utils;
```

Comment:
@jaggernod, I couldn't find any license information for this file, just some unlicensed gists. Would you have any ideas of source with a license declaration?